clox/functions.py

```diff
@@ -125,13 +125,30 @@ def show_countries_list() -> None:
                                                                     country_code=country_code, country_name=country_code))
 
 
+def _get_weekday_id(first_weekday: str, date_system: str = "gregorian") -> int:
+    """
+    Get weekday id.
+
+    :param first_weekday: first weekday
+    :param date_system: date system
+    """
+    first_weekday_normalized = first_weekday
+    if len(first_weekday) > 2:
+        first_weekday_normalized = first_weekday[:2].upper()
```

Comment:
```python
first_weekday_normalized = first_weekday[:2]
```

Reply:
I believe you mean `first_weekday_normalized = first_weekday_normalized[:2]`, I updated it.